LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
Oh! That's a bug. In that case, we should fix it elsewhere too.
```
$ git grep tspace | grep maxlen
LA4Falcon.c:        tp_wide = Number_Digits((int64) db1->maxlen/tspace+2);
LA4Falcon.c:        tp_wide = Number_Digits((int64) db2->maxlen/tspace+2);
LA4Ice.c:        tp_wide = Number_Digits((int64) db1->maxlen/tspace+2);
LA4Ice.c:        tp_wide = Number_Digits((int64) db2->maxlen/tspace+2);
LAshow.c:        tp_wide = Number_Digits((int64) db1->maxlen/tspace+2);
LAshow.c:        tp_wide = Number_Digits((int64) db2->maxlen/tspace+2);
```
I'll submit the `LAshow.c` fix to the upstream repo when I have time.